afs-ext-base/src/main/java/com/powsybl/afs/ext/base/VirtualCase.java

```diff
@@ -69,6 +70,11 @@ public Network getNetwork() {
         return findService(NetworkCacheService.class).getNetwork(this);
     }
 
+    @Override
+    public Network getNetwork(NetworkListener listener) {
```

Comment:
We must add some unit tests : 
maybe add a unit test for virtual cases to check that the listeners are indeed notified of modifications applied by the script (is it your use case?).

Reply:
ok